pyvlx/connection.py

```diff
@@ -52,9 +53,13 @@ def data_received(self, data: bytes) -> None:
         while self.tokenizer.has_tokens():
             raw = self.tokenizer.get_next_token()
             assert raw is not None
-            frame = frame_from_raw(raw)
-            if frame is not None:
-                self.frame_received_cb(frame)
+
+            try:
+                frame = frame_from_raw(raw)
+                if frame is not None:
+                    self.frame_received_cb(frame)
+            except Exception:
```

Comment:
Cant we catch a more specialized Exception here?

Reply:
OK, let's start catching PyVLXException. I think the intention was to avoid that corrupt data crashes pyvlx.